index.json

```
@@ -78,5 +78,246 @@
+        "enAlususAnswer": "Source/ArrayRelatedOperations/erase.en.alusus",
+        "arAlususAnswer": "Source/ArrayRelatedOperations/erase.ar.alusus"
+    },
+    {
+        "enPrompt": "Write Floyd Warshall algorithm",
+        "arPrompt": "اكتب خوارزمية Floyd Warshall",
+        "cAnswer": "Source/GraphAlgorithms/floyd_warshall.c",
+        "jsAnswer": "Source/GraphAlgorithms/floyd_warshall.js",
+        "enAlususAnswer": "Source/GraphAlgorithms/floyd_warshall.en.alusus",
+        "arAlususAnswer": "Source/GraphAlgorithms/floyd_warshall.ar.alusus"
+    },
+    {
+        "enPrompt": "check if a character is from the alphabet",
+        "arPrompt": "اختبر فيما إذا كان محرف من الأبجدية",
+        "cAnswer": "Source/StringRelatedOperations/is_alpha.c",
+        "jsAnswer": "Source/StringRelatedOperations/is_alpha.js",
+        "enAlususAnswer": "Source/StringRelatedOperations/is_alpha.en.alusus",
+        "arAlususAnswer": "Source/StringRelatedOperations/is_alpha.ar.alusus"
+    },
+    {
+        "enPrompt": "check if a character is a digit",
+        "arPrompt": "اختبر فيما إذا كان محرف هو رقم بين 0 و 9",
+        "cAnswer": "Source/StringRelatedOperations/is_digit.c",
+        "jsAnswer": "Source/StringRelatedOperations/is_digit.js",
+        "enAlususAnswer": "Source/StringRelatedOperations/is_digit.en.alusus",
+        "arAlususAnswer": "Source/StringRelatedOperations/is_digit.ar.alusus"
+    },
+    {
+        "enPrompt": "check if a string is alphanumeric",
+        "arPrompt": "اختبر فيما إذا كانت كل محارف سلسلة نصية من الأبجدية أو الأرقام",
+        "cAnswer": "Source/StringRelatedOperations/is_string_alphanumeric.c",
+        "jsAnswer": "Source/StringRelatedOperations/is_string_alphanumeric.js",
+        "enAlususAnswer": "Source/StringRelatedOperations/is_string_alphanumeric.en.alusus",
+        "arAlususAnswer": "Source/StringRelatedOperations/is_string_alphanumeric.ar.alusus"
+    },
+    {
+        "enPrompt": "check if a string contains a given char",
+        "arPrompt": "اختبر فيما إذا كانت سلسلة نصية تحتوي محرف معطى",
+        "cAnswer": "Source/StringRelatedOperations/char_exists.c",
+        "jsAnswer": "Source/StringRelatedOperations/char_exists.js",
+        "enAlususAnswer": "Source/StringRelatedOperations/char_exists.en.alusus",
+        "arAlususAnswer": "Source/StringRelatedOperations/char_exists.ar.alusus"
+    },
+    {
+        "enPrompt": "check if an array contains a given element",
+        "arPrompt": "اختبر فيما إذا كانت مصفوفة تحتوي على عنصر معطى",
+        "cAnswer": "Source/ArrayRelatedOperations/exists.c",
+        "jsAnswer": "Source/ArrayRelatedOperations/exists.js",
+        "enAlususAnswer": "Source/ArrayRelatedOperations/exists.en.alusus",
+        "arAlususAnswer": "Source/ArrayRelatedOperations/exists.ar.alusus"
+    },
+    {
+        "enPrompt": "find the position of a given char in a string",
+        "arPrompt": "أوجد موقع محرف معطى ضمن سلسلة نصية",
```

```suggestion
        "arPrompt": "جد موقع محرف معطى ضمن سلسلة نصية",
```

أوجد لا تأتي بمعنى البحث، وإنما بمعنى جعل الشيء موجودا.